apache-hadoop-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,155 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'apache-hadoop',
+        rules: [
+          {
+            alert: 'ApacheHadoopLowHDFSCapacity',
+            expr: |||
+              min without(job, name) (100 * hadoop_namenode_capacityremaining / clamp_min(hadoop_namenode_capacitytotal, 1)) < %(alertsWarningHDFSCapacity)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'Remaining HDFS cluster capacity is low which may result in DataNode failures or prevent DataNodes from writing data.',
+              description:
+                (
+                  '{{ printf "%%.0f" $value }} percent remaining HDFS usage on {{$labels.hadoop_cluster}} - {{$labels.instance}}, ' +
+                  'which is below the threshold of %(alertsWarningHDFSCapacity)s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'ApacheHadoopHDFSMissingBlocks',
+            expr: |||
+              max without(job, name) (hadoop_namenode_missingblocks) > %(alertsCriticalHDFSMissingBlocks)s
+            ||| % $._config,
+            'for': '1m',
```

Comment:
It is better to increase for at least 5m to avoid flapping alerts.

Reply:
👍 updated